lib/openstudio/workflow/run.rb

```diff
@@ -241,6 +241,12 @@ def run
         ensure
           @logger.info 'Workflow complete'
 
+          # If we let the :reporting_measures step fail to continue with
+          # postprocess, we still want to report and error
+          if @final_message != ''
+            @current_state = :errored
+          end
+
```

Comment:
Actual change one. I'm using @final_message as a test for the workaround below, so we still output a failure